books/free-programming-books-hi.md

```diff
@@ -23,6 +25,13 @@
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
 
 
+### Computer architecture
+
+* [कम्प्यूटर सिस्टम आर्किटेक्चर](https://www.aicte-india.org/sites/default/files/HINDI_BOOKS/BOOK%207.pdf) - डॉ. एस. एस. श्रीवास्तव, उच्च शिक्षा उत्कृष्टता संस्थान, भोपाल (म. प्र. ) (PDF)
+* [Computer architecture and organization notes](https://drive.google.com/file/d/1aE0mIBpmjansb9onxpw4Fh9xfGYPI96B/view) - eHindiStudy.com
```

Comment:
In general, we do not link to file sharing sites, or file hosting sites such as google docs. or dropbox. remove it